packages/reakit/src/Popover/PopoverState.ts

```diff
@@ -124,11 +128,12 @@ export function usePopoverState(
     unstable_flip: flip = true,
     unstable_shift: shift = true,
     unstable_inner: inner = false,
+    unstable_offset: offset = [0, gutter],
     unstable_preventOverflow: preventOverflow = true,
     unstable_boundariesElement: boundariesElement = "scrollParent",
     unstable_fixed: fixed = false,
     ...sealed
-  } = useSealedState(initialState);
+  }: PopoverInitialState = useSealedState(initialState);
```

Comment:
The redundant type annotation is required because otherwise typescript complains at line 126: "TS7022: `gutter` implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer".

An alternative would be to not set a default for unstable_offset as a prop, and instead do something like

```ts
// In the state destructuring
unstable_offset: offset,

// Later in the component
const finalOffset = unstable_offset ?? [0, gutter];
```